debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -156,9 +156,8 @@ private String tableSchemaName(TableId tableId) {
         else if (Strings.isNullOrEmpty(tableId.schema())) {
             return tableId.catalog() + "." + tableId.table();
         }
-        // When both catalog and schema is present then only schema is used
         else {
-            return tableId.schema() + "." + tableId.table();
+            return tableId.catalog() + "." + tableId.schema() + "." + tableId.table();
```

Comment:
to get db aware schema names like `server1.testDB.dbo.tablea.Value`

Reply:
For now, it's fine but I'm not sure what the result will look like for MySQL. `SCHEMA` is a [synonym](https://dev.mysql.com/doc/refman/8.0/en/create-database.html) of a `DATABASE` there. Using both pieces unconditionally may produce the results like `db.db.table`.